src/grandorgue/GOOrganController.cpp

```diff
@@ -118,10 +121,15 @@ GOOrganController::GOOrganController(
     m_MidiSamplesetMatch(),
     m_SampleSetId1(0),
     m_SampleSetId2(0),
-    m_bitmaps(this),
+    m_bitmaps(nullptr),
     m_PitchLabel(this),
     m_TemperamentLabel(this),
     m_MainWindowData(this, wxT("MainWindow")) {
+  if (isAppInitialized) {
+    // Load here objects that needs App (wx) to be loaded
+    m_TimerManager = new GOTimer();
```

Comment:
If you create these objects here you should also delete these objects in ~GOOrganController

Reply:
Of course. Too used to python where it is not necessary :sweat_smile: